docker-compose.yml

```diff
@@ -1,11 +1,13 @@
-jackrabbit:
-  image: node
-  working_dir: /code
-  volumes:
-    - .:/code
-  links:
-    - rabbit
-  environment:
-    RABBIT_URL: 'amqp://rabbit'
-rabbit:
-  image: rabbitmq
+version: '3'
```

Comment:
Is there any reason to require version 3? Won't this existing file work OK on version 3 too? If so, let's leave as is.